Makefile

```diff
@@ -28,31 +32,45 @@ lint:
 	)
 
 test:
-	pytest tests
-
-test-all:
-	tox
+	python -m pytest tests
```

Comment:
I think we want to exclude the beacon tests here? They need some special setup so I'm getting a bunch of fails. I can see the argument either way here though